onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
That sounds like a pretty good solution to me.  We'd have to worry about updating the hierarchy tree during mesos releases, but it not a difficult task.